types/index.ts

```diff
@@ -81,42 +81,42 @@ export interface Report {
  * @property {string} uri - The URI of the claim metadata.
  */
 export interface Claim {
-	__typename?: "Claim";
-	contract: string;
-	// biome-ignore lint: type definition imported from @hypercerts-org/sdk
-	tokenID: any;
-	// biome-ignore lint: type definition imported from @hypercerts-org/sdk
-	creator?: any | null;
-	id: string;
-	// biome-ignore lint: type definition imported from @hypercerts-org/sdk
-	owner?: any | null;
-	// biome-ignore lint: type definition imported from @hypercerts-org/sdk
-	totalUnits?: any | null;
-	uri?: string | null;
+  __typename?: "Claim";
+  contract: string;
+  // biome-ignore lint: type definition imported from @hypercerts-org/sdk
+  tokenID: any;
+  // biome-ignore lint: type definition imported from @hypercerts-org/sdk
+  creator?: any | null;
+  id: string;
+  // biome-ignore lint: type definition imported from @hypercerts-org/sdk
+  owner?: any | null;
+  // biome-ignore lint: type definition imported from @hypercerts-org/sdk
+  totalUnits?: any | null;
```

Comment:
What up with all the any types?

Reply:
not sure, @baumstern, can you look into this?